Create ui-c8y-1018-503-52-crl-c-8-y-help-5440-graft-release-y-2024-5487.md (#1754)
SoftwareAG/c8y-docs: content/change-logs/ui-c8y-1018-503-52-crl-c-8-y-help-5440-graft-release-y-2024-5487.md

---
date: ""

> title: crl c8y-help (#5440) [GRAFT][release/y2024] (#5487)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Help Component Update
```

> title: Additional information about certificate settings page.

Reviewer: ```suggestion
title: Added information about certificate management
```

> product_area: Platform services

Reviewer: Should be "Device management app", shouldn´t it @janhommes ?

Reviewer 2: No, guess this is with A&A. Can you please ping the developer @mmed-softwareag? You can always get the contributor from the PR. I can only do the reviews for the AppEnablement team.

Reviewer 3: correct, this is part of trusted certificates in the Device management app.

Reviewer 3: changed.

change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:

>   - value: q3kclF6pO

Reviewer: ```suggestion
  - value: component--KIsStyzM 
```

>     label: Authentication

Reviewer: Should probably be "Device Management app"

>     label: Authentication

Reviewer: ```suggestion
    label: Device Management app
```

build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-57827
version: 1018.503.52
---

> crl c8y-help (#5440) [GRAFT][release/y2024] (#5487)

Author: The Claude AI suggested to use the following body:
```suggestion
The help component has been updated in this release. This update includes improvements to the search functionality, making it easier for users to find the information they need. A few minor bug fixes and optimizations have also been implemented to improve performance and stability of the help system.
```

> The information regarding certificate management along with a link to the documentation is available on the certificate settings page.

Reviewer: ```suggestion
Information about certificate management along with a link to the documentation has been added on the **Settings** tab of the **Trusted certificates** page.
```

Reviewer: Or does the information show up on the **Trusted certificates** page in general?
